server/index.js

```diff
@@ -1,20 +1,63 @@
+/* eslint-disable func-names */
 const express = require('express');
 
 const app = express();
 const bodyPraser = require('body-parser');
 const db = require('./db/controller.js');
 
-const PORT = 3030;
+const port = process.env.PORT || 3030;
 
 app.use(express.static('client/dist/'));
 app.use(bodyPraser.json());
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// let resultHandler = function (err, data) {
+//   if (err) {
+//     res.status(500);
+//     res.send(err);
+//   } else {
+//     res.status(200);
+//     res.send(data);
+//   }
+// };
+
 app.get('/client', (req, res) => {
   res.send('Welcome to the client hacker!');
 });
 
 app.get('/reviews', (req, res) => {
-  db.getAll((err, data) => {
+  db.getReviewItems((err, data) => {
+    if (err) {
+      res.status(500);
+      res.send(err);
+    } else {
+      res.status(200);
+      res.send(data);
+    }
+  });
+});
+
+
+app.get('/attributesrating/:attribute?', (req, res) => {
+  db.getRatingAttributes((err, data) => {
+    if (err) {
+      res.status(500);
+      res.send(err);
+    } else {
+      res.status(200);
+      let result;
+      if (req.params.attribute) {
```

Comment:
I like how you handled this condition to get the correct data.

Reply:
Thank you!